arch/arm64/boot/dts/qcom/sdm636-motorola-beckham.dts

```diff
@@ -0,0 +1,744 @@
+			record-size = <0x3f800>;
+			ftrace-size = <0x0>;
+			pmsg-size = <0x40000>;
+		};
+
+		removed_region: removed_region@85800000 {
+			reg = <0x0 0x85800000 0x0 0x3700000>;
+			no-map;
+		};
+
+		adsp_mem: adsp-region@f6c00000 {
+			reg = <0x0 0xf6c00000 0x0 0x800000>;
+			no-map;
+		};
+
+		qseecom_mem: qseecom-region@f5800000 {
+			reg = <0x0 0xf5800000 0x0 0x1400000>;
+			no-map;
+		};
+
+		rmtfs_mem: memory@fd401000 {
+			compatible = "qcom,rmtfs-mem";
+			reg = <0x0 0xfd401000 0x0 0x200000>;
+			no-map;
+
+			qcom,client-id = <1>;
+			qcom,vmid = <QCOM_SCM_VMID_MSS_MSA>;
+		};
+	};
+
+	vph_pwr: vph-pwr-regulator {
+		compatible = "regulator-fixed";
+		regulator-name = "vph_pwr";
+		regulator-min-microvolt = <3700000>;
+		regulator-max-microvolt = <3700000>;
+
+		regulator-always-on;
+		regulator-boot-on;
+	};
+};
+
+&adreno_gpu {
+	status = "okay";
+};
+
+&adreno_gpu_zap {
+	firmware-name = "a512_zap.mdt";
+};
+
+&adsp_pil {
+	firmware-name = "adsp.mdt";
+};
+
+&blsp2_uart1 {
+	status = "okay";
+
+	/* HCI Bluetooth */
+	bluetooth {
+		compatible = "qcom,wcn3990-bt";
+		vddxo-supply = <&vreg_l9a_1p8>;
+		vddrf-supply = <&vreg_l6a_1p3>;
+		vddch0-supply = <&vreg_l19a_3p3>;
+		vddio-supply = <&vreg_l13a_1p8>;
+		max-speed = <3200000>;
+	};
+};
+
+&blsp_i2c2 {
+	status = "okay";
+
+	gpio_expander_1: gpio@20 {
+		compatible = "nxp,pcal6416";
+		reg = <0x20>;
+
+		gpio-controller;
+		#gpio-cells = <2>;
+
+		vcc-supply = <&vreg_l13a_1p8>;
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&gpio_expander_default>;
+
+		reset-gpios = <&tlmm 34 GPIO_ACTIVE_LOW>;
+	};
+
+	gpio_expander_2: gpio@21 {
+		compatible = "nxp,pcal6416";
+		reg = <0x21>;
+
+		gpio-controller;
+		#gpio-cells = <2>;
+
+		vcc-supply = <&vreg_l13a_1p8>;
+/*
+		Both GPIO expanders share a reset pin, Linux can't handle this as far as I know
+		pinctrl-names = "default";
+		pinctrl-0 = <&gpio_expander_default>;
+
+		reset-gpios = <&tlmm 34 GPIO_ACTIVE_LOW>;
+*/
+	};
+	/* ti,drv2624 @ 0x5a, vibrator */
+	/* analogix,anx7805 @ 0x72 hdmi converter for mods */
+};
+
+&blsp_i2c4 {
+	status = "okay";
+
+	touchscreen@20 {
+		compatible = "syna,rmi4-i2c";
+		reg = <0x20>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		interrupt-parent = <&tlmm>;
+		interrupts = <67 IRQ_TYPE_EDGE_FALLING>;
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&ts_reset>;
+
+		vdd-supply = <&vreg_l6b_3p3>;
+		vio-supply = <&vreg_l11a_1p8>;
+
+		rmi4-f01@1 {
+			reg = <0x01>;
+			syna,nosleep-mode = <1>;
+		};
+
+		rmi4-f12@12 {
+			reg = <0x12>;
+			syna,sensor-type = <1>;
+		};
+	};
+};
+
+&blsp_i2c6 {
+	status = "okay";
+
+	nfc@28 {
+		compatible = "nxp,nxp-nci-i2c";
+		reg = <0x28>;
+
+		interrupt-parent = <&tlmm>;
+		interrupts = <40 IRQ_TYPE_EDGE_RISING>;
+
+		enable-gpios = <&tlmm 35 GPIO_ACTIVE_HIGH>;
+		firmware-gpios = <&tlmm 61 GPIO_ACTIVE_HIGH>;
+
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&nfc_default>;
+	};
+};
+
+&mdss {
+	status = "okay";
+};
+
+&mdss_dsi0 {
+	status = "okay";
```

Comment:
Accordin to lastest trends in upstream DTS styling, status should be last property (but come before `panel: panel@0 {` of course)